README.md

```diff
@@ -1,20 +1,14 @@
 <!--
- Licensed to the Apache Software Foundation (ASF) under one
- or more contributor license agreements.  See the NOTICE file
- distributed with this work for additional information
- regarding copyright ownership.  The ASF licenses this file
+ Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.  See the NOTICE file distributed with this work for additional information regarding copyright ownership.  The ASF licenses this file
```

Comment:
Airflow has identical license info at the top of every file, and that should stay consistent. Can you revert the newlines and line length back to the original format?